app/src/main/kotlin/me/echeung/moemoekyun/ui/screen/home/Player.kt

```diff
@@ -109,17 +123,16 @@ fun PlayerScaffold(
         sheetContent = {
             PlayerContent(
                 radioState = radioState,
+                playPauseButtonState = playPauseButtonState,
                 accentColor = accentColor,
                 onClickStation = onClickStation,
                 onClickHistory = onClickHistory,
-                togglePlayState = togglePlayState,
                 toggleFavorite = toggleFavorite,
-                onClickCollapse = {
-                    scope.launch {
-                        scaffoldState.bottomSheetState.hide()
-                    }
-                },
-            )
+            ) {
```

Comment:
I actually prefer keeping the explicit argument if it's not actually a Composable content block.